build.sbt

```diff
@@ -21,4 +32,65 @@ lazy val docs = (project in file("docs"))
   .enablePlugins(BuildInfoPlugin)
 
 addCommandAlias("ci-test", "scalafmtCheck; scalafmtSbtCheck; docs/tut; compile; test; scripted")
-addCommandAlias("ci-docs", "docs/tut")
+addCommandAlias("ci-docs", "docs/mdoc; headerCreateAll")
```

Comment:
I was wondering why we use `docs/tut` in the `ci-test` alias and `docs/mdoc` in `ci-docs`

Reply:
Fixed in d7f7aa2